build/trend-scenarios.yml

```diff
@@ -91,6 +91,13 @@ parameters:
   - displayName: Json 10k
     arguments: --scenario json_middleware --variable singleStringJsonSize=10000 --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/json.benchmarks.yml --property scenario=Json10k
 
+# Antiforgery
+
+  - displayName: Antiforgery Token Validation
```

Comment:
I think `scenario=...` needs to be a unique name, we use that name for the checkboxes on the perf dashboard.

Reply:
Yep, thanks, I changed to unique `--property scenario=...`